src/main/java/edu/ucsb/cs156/courses/controllers/ApiController.java

```diff
@@ -28,7 +29,7 @@ protected Object genericMessage(String message) {
     return Map.of("message", message);
   }
 
-  @ExceptionHandler({ EntityNotFoundException.class, BadEnrollCdException.class, CharLimitExceededException.class })
+  @ExceptionHandler({ EntityNotFoundException.class, BadEnrollCdException.class, CharLimitExceededException.class, NameAndQuarterExistsException.class })
```

Comment:
I won't hold up the PR for this... but I'll instead give you an opportunity for another small 5 point issue.

It might be better to separate this exception handler into two.   The `@ResponseStatus(HttpStatus.NOT_FOUND)` (404) is really only appropriate for an  `EntityNotFoundException`, and marginally for a `BadEnrollCdException`.

It would be better to use `BAD_REQUEST` (400) for instances of `CharLimitExceededException`, `NameAndQuarterExistsException`.

You accomplish this by copying/pasting the entire method, and just having different values for the parameters to the `@ResponseStatus` `@ExceptionHandler` annotations.

If you'd like to add that to your backlog for a quick win of 5 more points, please do.  (I think this is a 5 point issue,but if it ends up being more complicated that I anticipate, I'm willing to consider 10).